config.example.cfg

```diff
@@ -1,6 +1,6 @@
 [pipeline]
-plate_path = ~/Cellos/cellos_data/
-output_path = ~/Cellos/output/
+plate_path = ../../cellos_data/
```

Comment:
These are relative to the scripts, using the repo layout. This way things run following the instructions. I did make a note that someone could use absolute paths here to the actual data as needed